treebeard-lib/treebeard/conf.py

```diff
@@ -44,6 +44,8 @@ def get_deglobbed_notebooks(self):
         deglobbed_notebooks = []
         for pattern in self.notebooks:
             deglobbed_notebooks.extend(sorted(glob(pattern, recursive=True)))
+        if len(deglobbed_notebooks) == 0:
+            raise Exception("No notebooks found in project! (**/*.ipynb)")
```

Comment:
let's test this by removing the yaml file from one of our examples or integ tests